internal/condition/metrics/metrics.go

```diff
@@ -0,0 +1,72 @@
+// Packge metrics provides various metric and telemetry definitions for OpenFGA Conditions.
+package metrics
+
+import (
+	"time"
+
+	"github.com/openfga/openfga/internal/build"
+	"github.com/openfga/openfga/internal/server/config"
+	"github.com/openfga/openfga/internal/utils"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	subsystem = "conditions"
+)
+
+// Metrics provides access to Condition metrics.
+var Metrics *ConditionMetrics
+
+func init() {
+	m := &ConditionMetrics{
+		compilationTime: promauto.NewHistogram(prometheus.HistogramOpts{
+			Namespace: build.ProjectName,
+			Subsystem: subsystem,
+			Name:      "condition_compilation_duration_ms",
+			Help:      "A histogram measuring the compilation time (in milliseconds) of a Condition.",
+			Buckets:   []float64{1, 5, 15, 50, 100, 250, 500, 1000},
+		}),
+
+		evaluationTime: promauto.NewHistogram(prometheus.HistogramOpts{
+			Namespace: build.ProjectName,
+			Subsystem: subsystem,
+			Name:      "condition_evaluation_duration_ms",
+			Help:      "A histogram measuring the evaluation time (in milliseconds) of a Condition.",
+			Buckets:   []float64{0.1, 0.25, 0.5, 1, 5, 15, 50, 100, 250, 500},
```

Comment:
We can tune these overtime as we see more practical distributions of them, but given the benchmarks we've already done around evaluation I thought this was a decent distribution range.